infrahub_sdk/client.py

```diff
@@ -623,6 +628,7 @@ async def filters(
         prefetch_relationships: bool = ...,
         partial_match: bool = ...,
         property: bool = ...,
+        batch: bool = ...,
```

Comment:
Thinking out loud
I'm not sure if `batch` will be clear enough for the end user ? 
As an alternative we could use `parallel`
any preference @opsmill/customer-success ?

Reply:
I have renamed to `parallel` for now.